src/plugins/intel_cpu/src/ngraph_transformations/convert_fq_rnn_to_quantized_rnn.cpp

```diff
@@ -0,0 +1,226 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "convert_fq_rnn_to_quantized_rnn.hpp"
+
+#include <cstdlib>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include <ngraph/rt_info.hpp>
+#include "ie_common.h"
+#include "ngraph/except.hpp"
+#include "ngraph/node_output.hpp"
+#include "ngraph/op/constant.hpp"
+#include "ngraph/op/convert.hpp"
+#include "ngraph/op/fake_quantize.hpp"
+#include "ngraph/op/lstm_cell.hpp"
```

Comment:
Looks like we can avoid these includes (please see the next comment)

Reply:
refactored to use opset9